corehq/apps/hqwebapp/views.py

```diff
@@ -862,13 +862,14 @@ class SolutionsFeatureRequestView(View):
     urlname = 'solutions_feature_request'
 
     def post(self, request, *args, **kwargs):
-        email = _get_email_message_base(
-            post_params=request.POST,
-            couch_user=request.couch_user,
-            uploaded_file=request.FILES.get('feature_request'),
-            to_email=settings.INTERNAL_FEEDBACK_EMAIL,
-        )
-        email.send(fail_silently=False)
+        if request.couch_user.is_staff:
```

Comment:
should we return a 400 or similar in case the user isn't a staff?

Reply:
Good idea, this would make it more obvious that something went wrong.

Reply:
[df1e692](https://github.com/dimagi/commcare-hq/pull/34026/commits/df1e6928086376179dab184076e138ab9612fb9b)